### ISSUE TITLE
Launch Agents & Daemons

### ISSUE THREAD

**Issue by pbGuy** [alterstep/dnscrypt-osxclient#111]
![etrecheck 2016-09-01](https://cloud.githubusercontent.com/assets/7810315/18173220/ac604f6a-701d-11e6-896a-c72c12f7302a.png)

(All the apps, shown in the screen shot, are using most recent update. Using OS 10.11.6.)

Using the above screen shot (a partial from running EtreCheck on 9/1/16), I am wondering if the items related to DNSCrypt 1.0.12 are telling me about some issue(s), particularly the item that shows it _**failed**_ to load. 

[Setting this EtreCheck aside, DNSCrypt 1.0.12, set to using resolver from Cisco over UDP, seems to be running Ok.  ![finder partial menu bar](https://cloud.githubusercontent.com/assets/7810315/18173825/fa3a2b82-701f-11e6-9baf-fd43f1e7d620.png)  At least, I've not noticed any issues.]

Also, is there some issue whereby the DNSCrypt OSX Client is showing a report of _**Invalid Signature**_? 

Do I need to take some corrective action regarding my installation of DNSCrypt 1.0.12?

**Comment by ajkblue** [comment]
I have also noticed this, running LaunchControl I have found errors in "com.github.dnscrypt-osxclient.DNSCryptAfterboot" and "com.github.dnscrypt-osxclient.DNSCryptConsoleChange", stating that "This job has failed for no apparent reason". I believe that these have been longstanding issues, as I have seen these errors in past versions of the DNSCrypt client.
![screen shot 2016-09-04 at 11 17 16 pm](https://cloud.githubusercontent.com/assets/9651939/18238044/e9c2684a-72f5-11e6-9aff-d1a3e401ac90.png)
I am also wondering, is there a way to fix these issues? Is there any cause for concern here?

**Comment by pbGuy** [comment]
Since my prior post, I've now upgraded my OS to macOS 10.12. Today (9/30/16), I updated EtreCheck to v3.0.6 and received the following report:

![etrecheck 9-30-16](https://cloud.githubusercontent.com/assets/7810315/18994769/2e8b20ce-86de-11e6-9e64-05c848a878b8.png)

The DNSCrypt Afterboot plist has loaded. While I'm not sure about the plist's successful loading, it's an improvement over my prior snapshot, which showed the Afterboot plist did not load.
